### DISCUSSION TITLE
react-hook-form has sideEffects: true

### DISCUSSION THREAD

**Discussion by alexey-ogarkov-db** [react-hook-form/react-hook-form#8038] [Q&A]
Hi Guys, is there any particular reason to have sideEffects: true for this project?

**Comment by bluebill1049** [comment]
it shouldn't, feel free to send us a PR to correct that.

**Reply by alexogar** [reply]
Hey cool raised https://github.com/react-hook-form/react-hook-form/pull/8041

**Reply by alexey-ogarkov-db** [reply]
Is it possible to have a release with that?

**Reply by Moshyfawn** [reply]
https://github.com/react-hook-form/react-hook-form/pull/8041#issuecomment-1073004101
